internal/webserver/handler-api.go

```diff
@@ -108,6 +108,16 @@ func (h *handler) apiLogin(w http.ResponseWriter, r *http.Request, ps httprouter
 		genSession(model.Account{
```

Comment:
This endpoint is no longer used in latest `master` so you can ignore it. Merge master into you branch to see latest changes.